src/frontend/src/constant/constant.ts

```diff
@@ -173,6 +173,22 @@ export const MinistryList = [
   },
 ];
 
+export const PiaStatusList = [
+  { label: 'Incomplete', value: 'INCOMPLETE' },
+  { label: 'Edit_In_Progress', value: 'EDIT_IN_PROGRESS' },
+  { label: 'MPO_Review', value: 'MPO_REVIEW' },
+  { label: 'PCT_Review', value: 'PCT_REVIEW' },
```

Comment:
We will most likely have to to change these labels to use spaces instead of underscores at some point in the future.